cloudfront.tf

```diff
@@ -0,0 +1,83 @@
+locals {
+  cf_origin_id = "host"
+}
+
+resource "aws_cloudfront_distribution" "geofencing" {
+  provider = aws.iam-Perimeter
+
+  
+  origin {
+    custom_origin_config {
+      http_port              = 80
+      https_port             = 443
+      origin_protocol_policy = "https-only"
+      origin_ssl_protocols = [
+      "TLSv1.2"]
+    }
+
+    domain_name = "https://startup-sample-project.tnfhhm-dev.nimbus.cloud.gov.bc.ca/"
+    origin_id   = "https://startup-sample-project.tnfhhm-dev.nimbus.cloud.gov.bc.ca/"
```

Comment:
you can use the local variable `cf_origin_id` and set it there. This just has to be a unique id, it can be anything.

Reply:
Made changes, Please reveiw